app/notification_email.go

```diff
@@ -320,7 +320,7 @@ func (a *App) GetMessageForNotification(post *model.Post, translateFunc i18n.Tra
 	// extract the filenames from their paths and determine what type of files are attached
 	infos, err := a.Srv.Store.FileInfo().GetForPost(post.Id, true, false, true)
 	if err != nil {
-		mlog.Warn(fmt.Sprintf("Encountered error when getting files for notification message, post_id=%v, err=%v", post.Id, err), mlog.String("post_id", post.Id))
+		mlog.Warn("Encountered error when getting files for notification message", mlog.String("post_id", post.Id) mlog.Err(err))
```

Comment:
```
# github.com/mattermost/mattermost-server/app
app/notification_email.go:323:110: syntax error: unexpected mlog, expecting comma or )
# github.com/mattermost/mattermost-server/app [github.com/mattermost/mattermost-server/app.test]
app/notification_email.go:323:110: syntax error: unexpected mlog, expecting comma or )
make: *** [Makefile:155: govet] Error 1
```

Reply:
Oops! Sorry about that, all fixed now @levb  :+1: